php/common.php

```diff
@@ -174,7 +176,8 @@ function get_settings($filename = "")
     // отчёты
     $config['reports'] = [
         'auto_clear_messages' => $ini->read('reports', 'auto_clear_messages', 0),
-        'exclude_forums_ids' => $ini->read('reports', 'exclude', ''),
+        'exclude_forums_ids' => $ini->read('reports', 'exclude_forum', ''),
```

Comment:
If I am not mistaken here, those who already have their excluded forum IDs under `reports -> exclude`, will lose the setting. This either should be marked as a breaking change or be handled by `implode`'ing old `exclude` with the new `exclude_forum` when `exclude` is not empty.

Reply:
explode > implode does not make sense, because the string `exclude_forums_ids` is collected by iterating through the array $forums.
If such behavior is very necessary, then you can come up with something of course.

Reply:
The thing I want to point here is that `exclude` is currently used by Web-TLO users and we have at least communicate this change. In ideal scenario: we need to have here a transparent handling of that old `exclude` field. I tried to propose a way how to implement it but my PHP knowledge turned out to be insufficient... :-(

Reply:
It would be better to introduce new setting due to #167

Reply:
> It would be better to introduce new setting due to #167 instead of repurposing existing one.

Thus, we will postpone these changes until #167?

Reply:
> The thing I want to point here is that `exclude` is currently used by Web-TLO users and we have at least communicate this change. In ideal scenario: we need to have here a transparent handling of that old `exclude` field. I tried to propose a way how to implement it but my PHP knowledge turned out to be insufficient... :-(

For now, `set config` and `read config` in differnt stuff. also current `exclude` option is not loaded in `common.php`. So for purpose of parse current `exclude` option I need first add it to config load, then parse it in UI settings, after that save and, probably, delete old `exclude` from `config.ini` (delete is unsupporned AFAIK).
Or do some backend magic, with close result.

I dont think, this is be too hard for our users to exclude new subsection fron UI after implement this PR.

Or, wait config refactor due #167, and after this add new options and functionality.

Reply:
Add migration with `user_version` bump https://github.com/keepers-team/webtlo/pull/194/commits/cde4e3916c0dc904d51b4e1bc9005d8c129d18af